src/c++/perf_analyzer/genai-perf/tests/test_cli.py

```diff
@@ -240,9 +241,10 @@ def test_file_flags_parsed(self, monkeypatch, mocker):
     def test_default_profile_export_filepath(
         self, monkeypatch, arg, expected_path, capsys
     ):
+        logging.init_logging()
```

Comment:
I know you have already created a ticket to make a fixture to run this before all tests.

I think that looks something like this:
`

    @pytest.fixture(autouse=True)

    def init_logging():

    logger.init_logging()

    yield

    (Any post test work you want to happen)
`

Reply:
Thanks for the suggestion! Yeah, there is a ticket. Unfortunately, because of the way capsys binds to sysout and the way our logger redirects stdout, it's more challenging than a fixture here. I spent some time today, but it's more involved (e.g. using caplog instead, giving our logger a testing mode, etc.). The ticket has more information.